src/d3a/d3a_core/redis_connections/redis_communication.py

```diff
@@ -97,6 +105,22 @@ def _generate_redis_response(self, response, simulation_id, is_successful, comma
                 })
         self.publish_json(response_channel, response_json)
 
+    def _area_map_callback(self, payload: Dict) -> None:
+        """Triggers the calculation of area uuid and name map and publish it
+        back to a redis response channel"""
+        area_map = self._area_uuid_name_map_wrapper(self._simulation.area)
```

Comment:
Nit, this could be renamed to `area_mapping` here and in the rest of occurrences 
```suggestion
        area_mapping = self._area_uuid_name_map_wrapper(self._simulation.area)
```